integration-testing/src/debugAgentTest/kotlin/DebugProbes.kt

```diff
@@ -1,14 +0,0 @@
-/*
- * Copyright 2016-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
- */
-@file:Suppress("INVISIBLE_MEMBER", "INVISIBLE_REFERENCE")
-package kotlin.coroutines.jvm.internal
```

Comment:
It actually wasn't necessary. It seemed to be working fine without this file, but that was only because I still had the compiled class file locally available. I have reverted the file back to it's original form.